### PR TITLE
update site

### PR DESCRIPTION
**Pull request by jhsiao999**

### CODE REVIEW

File: analysis/data-overview.Rmd [hunk 1]
```diff
@@ -1,36 +1,9 @@
 ---
 title: "Data overview"
 author: "Joyce Hsiao"
-output: 
-  html_document:
-    toc: TRUE
-    toc_float: FALSE
+output: workflowr::wflow_html
```
Comment (jdblischak) [inline analysis/data-overview.Rmd:4]: You can't update only one R Markdown file to use `workflowr::wflow_html`. It needs to be done for the whole project by editing `analysis/_site.yml` and also adding `site: workflowr::wflow_site` to `analysis/index.html`. Do you want me to run [`wflow_update()`](https://jdblischak.github.io/workflowr/reference/wflow_update.html) to update this project to use workflowr 1.0?

Comment (jhsiao999) [reply analysis/data-overview.Rmd:4]: I think it's a good idea to keep the site updated in general, so long as the update is not going to cost us too much time and efforts... What does `wflow_udpate` do? Is it going to recompile all the `html` files?

Comment (jdblischak) [reply analysis/data-overview.Rmd:4]: `wflow_update()` will only update the Rmd files and other settings files. After that you can decide how many of the HTML files you want to regenerate.

Comment (jhsiao999) [reply analysis/data-overview.Rmd:4]: Great. let's do that. I just didn't want to go into regenerating the `htmls`. I haven't gone back to all of them and am worried that some data/code links may break... Thanks!

Comment (jdblischak) [reply analysis/data-overview.Rmd:4]: OK. I'll merge this PR and then run `wflow_update()`. Then, each time you re-published an analysis, that HTML will have the workflowr 1.0 layout.

**Review by jdblischak — APPROVED** [review]

**Pull request merged**